qgis-app/styles/views.py

```diff
@@ -43,6 +47,92 @@ def check_styles_access(user, style):
     return False
 
 
+def send_mail_wrapper(subject,
+                      message,
+                      mail_from,
+                      recipients,
+                      fail_silently=True):
+    """
+    Wrapping send_email function to send email only when not DEBUG.
+    """
+
+    if settings.DEBUG:
+        logging.debug("Mail not sent (DEBUG=True)")
+    else:
+        send_mail(subject,
+                  message,
+                  mail_from,
+                  recipients,
+                  fail_silently)
+
+
+def style_notify(style, created=True):
+    """
+    Email notification when a new style created.
+    """
+    recipients = [u.email for u in User.objects.filter(
```

Comment:
So here I would prefer that we email specific users to let them know a new style has been added to the platform. I created the group 'Style Managers' for this purpose. Please make sure that we have documented in the README how to set up the style managers group and that the members of this group will receive style related emails.